Volume/Views/WeeklyDebriefView.swift

```diff
@@ -6,112 +6,251 @@
+                        })
+                            .padding([.leading, .trailing], 48)
+                        Spacer()
+                    }
+                    ForEach(debriefDisplayInfo.readArticles) { article in
+                        DebriefArticleView(header: "Share What You Read",
+                                           article: article,
+                                           debriefViewIsOpen: $isOpen,
+                                           didOpenURL: $openedURL,
+                                           showArticle: $onOpenArticleUrl)
+                    }
+                    ForEach(debriefDisplayInfo.randomArticles) { article in
+                        DebriefArticleView(header: "Top Articles of the Week",
+                                           article: article,
+                                           debriefViewIsOpen: $isOpen,
+                                           didOpenURL: $openedURL,
+                                           showArticle: $onOpenArticleUrl)
+                    }
+                    VStack {
+                        Header("See You Next Week!", .center)
+                            .padding(.top, 24)
                         VStack {
-                            Image("volume-logo")
-                                .resizable()
-                                .frame(width: 245, height: 75)
-                            Text("Stay updated with Cornell student publications, all in one place")
-                                .font(.begumRegular(size: 16))
-                        }
-                        .padding(.top, 200)
-                        Button {
-                            isOpen = false
-                        } label: {
-                            Text("Continue to Volume")
-                                .foregroundColor(Color.volume.orange)
-                                .font(.helveticaBold(size: 16))
+                            VStack {
+                                Image("volume-logo")
+                                    .resizable()
+                                    .frame(width: 245, height: 75)
+                                Text("Stay updated with Cornell student publications, all in one place")
+                                    .font(.begumRegular(size: 16))
+                            }
+                            .padding(.top, 200)
+                            Button {
+                                isOpen = false
+                            } label: {
+                                Text("Continue to Volume")
+                                    .foregroundColor(Color.volume.orange)
+                                    .font(.helveticaBold(size: 16))
+                            }
+                            .frame(width: 184, height: 45, alignment: .center)
+                            .overlay(RoundedRectangle(cornerRadius: 26)
+                            .stroke(Color.volume.orange, lineWidth: 2))
```

Comment:
Might be able to remove the `Color` here too.

Reply:
That results in getting an error `Type 'ShapeStyle' has no member 'volume'`